### DISCUSSION TITLE
Integrating just one requirement into an existing custom site, for free?

### DISCUSSION THREAD

**Discussion by inspector71** [tailwindlabs/tailwindcss#8928] [Help]
Looking for a trivial way to replace jQuery UI Tabs in an existing, custom site.

That tailwind apparently does not have a default style sheet / theme appeals as almost every other _bags of text_ out there enforce default styles that break existing sites. 

[Shoelace](https://shoelace.style/components/tab-group) but it seems possibly over-engineered and non-trivial to do something as simple as centering the tabs which is not an option so manual work is required. Ironically perhaps, it's default appearance does not need to interfere with existing styles, because it's a jailed Web Component, but being a component also stops it inheriting the rules I actually want it to. A Web Component paradox?

Is there a Tailwind tabs component and is it free? Most of the 'components' - maybe all of them -appear to be 'payware components' or am I misreading? The way the documentation tries to transparently jump to paid options to get people paying is, if I may say so, disingenuous.

**Comment by wongjn** [comment]
Tailwind itself is a CSS framework, not a UI library. It's a step lower than a UI library, since you can use Tailwind to to build UI components. The components you saw are probably from Tailwind UI, which is separate from the core of Tailwind. Tailwind UI **is** a UI library built from Tailwind. If you don't want to pay for Tailwind UI, you can always roll your own tab component using the Tailwind CSS framework.
